contracts/src/starknet/rewaave/tokens/ETHstaticAToken.cairo

```diff
@@ -128,6 +110,34 @@ end
 # Externals
 #
 
+@external
+func initialize_ETHstaticAToken{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
```

Comment:
why not simply `initialize`?

Reply:
I added the suffix to all the initialize functions otherwise their selector would clash with the one from the proxy